graphene_django/converter.py

```diff
@@ -96,7 +96,12 @@ class EnumWithDescriptionsType:
         def description(self):
             return str(named_choices_descriptions[self.name])
 
-    return_type = Enum(name, list(named_choices), type=EnumWithDescriptionsType)
+    return_type = Enum(
+        name,
+        list(named_choices),
+        type=EnumWithDescriptionsType,
+        description="An enumeration.",  # Temporary fix until https://github.com/graphql-python/graphene/pull/1502 is merged
```

Comment:
We can do `description=__ENUM_DESCRIPTION` here with

```py
import sys
__ENUM_DESCRIPTION = "An enumeration." if (sys.version_info.major, sys.version_info.minor) == (3, 11) else None
```

up top so behaviors for other Python versions won't change.